frontends/web/src/components/icon/icon.tsx

```diff
@@ -80,6 +79,11 @@ export const Info = (props: IconProps) => (<img src={info} draggable={false} {..
 export const BitBox02Stylized = (props: IconProps) => (<img src={BB02Stylized} draggable={false} {...props} />);
 export const ArrowDown = (props: IconProps) => (<img src={arrowDownSVG} draggable={false} {...props} />);
 export const Check = (props: IconProps) => (<img src={checkSVG} draggable={false} {...props} />);
-export const Checked = (props: IconProps) => (<img src={checkedSVG} draggable={false} {...props} />);
 export const Cancel = (props: IconProps) => (<img src={cancelSVG} draggable={false} {...props} />);
 export const Copy = (props: IconProps) => (<img src={copySVG} draggable={false} {...props} />);
+export const Checked = (props: IconProps) => (
```

Comment:
This is wrong now, since it is an inline SVG it should not use `JSX.IntrinsicElements['img']` but rather `JSX.IntrinsicElements['svg']`, will update the PR